checkov/sca_package/runner.py

```diff
@@ -162,6 +169,17 @@ def parse_vulns_to_records(
                     )
                 )
 
+    def get_license_status(self, license_dict, vulnerability):
```

Comment:
please add type hints here 🙂 probably something like that
```suggestion
    def get_license_status(self, license_dict: dict[tuple[str, str], dict[str, Any]], vulnerability: dict[str, Any]) -> str:
```